samples/wormhole/ping-pong/WormHole_1/GatewayA/app.config

```diff
@@ -0,0 +1,23 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
Are the assembly redirects necessary? Especially since that's the only thing in this app.config.

(Same in GatewayB)